### PR TITLE
Add C++ language stub, similar to Java

### PR DESCRIPTION
**Pull request by nwinter**

### CODE REVIEW

File: app/core/utils.coffee [hunk 1]
```diff
@@ -457,8 +457,9 @@ startsWithVowel = (s) -> s[0] in 'aeiouAEIOU'
 filterMarkdownCodeLanguages = (text, language) ->
   return '' unless text
   currentLanguage = language or me.get('aceConfig')?.language or 'python'
-  excludedLanguages = _.without ['javascript', 'python', 'coffeescript', 'clojure', 'lua', 'java', 'io', 'html'], currentLanguage
-  # Exclude language-specific code blocks like ```python (... code ...)``` for each non-target language.
+  excludedLanguages = _.without ['javascript', 'python', 'coffeescript', 'lua', 'java', 'cpp', 'html'], currentLanguage
```
Comment (AndrewJakubowicz) [inline app/core/utils.coffee]: Is there a possibility there is a `clojure` or `io` language that isn't excluded after this change?

File: app/views/play/level/tome/Spell.coffee [hunk 2]
```diff
@@ -81,6 +81,9 @@ module.exports = class Spell
           }
         }
       """
+    if @language is 'cpp' and not @languages[@language]
+      # TODO: do some transforms here to handle most common sample code differences between JavaScript and C++
+      @languages.cpp = @languages.java
```
Comment (AndrewJakubowicz) [inline app/views/play/level/tome/Spell.coffee]: What is this line doing?

File: app/core/utils.coffee (hunk 1 above)
Comment (nwinter) [reply app/core/utils.coffee]: I'll do a check and delete any if so.

File: app/views/play/level/tome/Spell.coffee (hunk 2 above)
Comment (nwinter) [reply app/views/play/level/tome/Spell.coffee]: It's a placeholder for any further automatic transforms we want to do. But yeah, I guess this should be coming from `javascript`.

**Pull request merged**